ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -35,6 +35,7 @@
   win_command: C:\temp\cygwin.exe -q -P "{{ item }}"
   with_items:
     - autoconf
+    - gcc-core
```

Comment:
I presume this role is still to be added? Doesn't seem to be part of this PR

Reply:
this just installs `gcc-core` into cygwin. It's not a role :)